ui/desktop/tests/integration/components/hidden-secret-test.js

```diff
@@ -1,23 +1,49 @@
 import { module, test } from 'qunit';
 import { setupRenderingTest } from 'ember-qunit';
-import { render, find } from '@ember/test-helpers';
+import { render, find, click } from '@ember/test-helpers';
+import setupMirage from 'ember-cli-mirage/test-support/setup-mirage';
 import { hbs } from 'ember-cli-htmlbars';
 import { setupIntl } from 'ember-intl/test-support';
-
 module('Integration | Component | hidden-secret', function (hooks) {
   setupRenderingTest(hooks);
   setupIntl(hooks);
+  setupMirage(hooks);
 
-  test('it renders with a secret', async function (assert) {
-    assert.expect(2);
-    await render(hbs`
-      <HiddenSecret @secret='AAaaBBccDDeeOTXzSMT1234BB_Z8JzG7JkSVxI' />
+  // Global Secret for mocking
+  this.secret = `{
+    "secret_key": "QWERTYUIOP",
+    "secret_access_key": "QWERT.YUIOP234567890",
+    "nested_secret": {
+      "session_token": "ZXCVBNMLKJHGFDSAQWERTYUIOP0987654321",
+      "complex_nest": {
+        "hash": "qazxswedcvfrtgbnjhyujm.1234567890"
+      }
+    }
+  }`;
+
+  async function renderComponent() {
+    return render(hbs`
+      <HiddenSecret
+        @secret={{this.secret}}
+      />
     `);
+  }
 
+  test('it renders with a secret', async function (assert) {
```

Comment:
Sorry I didn't catch this before but can we update the asserts to use `assert.dom()`. It will remove the need for `find()` and is more clear to future readers. 
`assert.dom('selector').exists();`
`assert.dom('selector').hasText(text);`

Reply:
+1 to this suggestion